server/Schemas/resolvers.js

```diff
@@ -1,12 +1,13 @@
 const { AuthenticationError } = require('apollo-server-express');
 const { User } = require('../models');
-const { signToken } = require('../utils/auth');
+const { signToken, authenticate } = require('../utils/auth');
```

Comment:
I created a authenticate method. Use this method only on mutations and queries that need to be authenticated/authorized.